sources/procfs.go

```diff
@@ -675,6 +675,31 @@ func parseFileElements(path string, directoryDepth int) (name string, nodeName s
 	return name, nodeName, nil
 }
 
+func convertToBytes(s string) string {
+	new_s := ""
+	multiplier := 1
+	switch finalChar := s[len(s)-1:]; finalChar {
+	case "K":
+		new_s = strings.TrimSuffix(s, "K")
+		multiplier = 1000
+	case "M":
+		new_s = strings.TrimSuffix(s, "M")
+		multiplier = 1000000
+	case "G":
+		new_s = strings.TrimSuffix(s, "G")
+		multiplier = 1000000000
+	default:
+		//this should only happen for integers
+		return s
+	}
+	byteVal, err := strconv.Atoi(new_s)
+	if err != nil {
+		return s
+	}
+	byteVal = byteVal * multiplier
+	return strconv.Itoa(byteVal)
```

Comment:
Ok, after looking at this again and looking at some of the float-to-string and string-to-float methods, I'm not sure that's any better. I could convert the math.Pow() operations to uint64 immediately, but that just makes the code messier. Any serious opposition to leaving the PR as-is currently?